### ISSUE TITLE
Error in ELI5 documentation?

### ISSUE THREAD

**Issue by felixsanz** [d8ahazard/sd_dreambooth_extension#1086]
![1679107378](https://user-images.githubusercontent.com/6182099/226079471-1705d68a-d0d2-4b47-8465-64a7bfe26028.png)

In the last line, where the generated class images are explained, this: `(2) Close up, bedroom background`, should not be `(2) cat, Close up, bedroom background` ? Why cat is not appended here? This ELI5 is kind of confusing xD

**Comment by ArrowM** [comment]
Thanks for the feedback! `(2) Close up, bedroom background` was actually a bug in the code. I just pushed a fix for it. Let me know if there are other parts of the ELI5 guide that are confusing.

**Comment by ArrowM** [comment]
Maybe I could add a line about this to the guide. The whole idea behind the `[filewords]` processing is that you can caption all of your images before you pick your instance and class tokens and then when you actually run the extension, it will handle redundant words.

**Comment by ArrowM** [comment]
I added a https://github.com/d8ahazard/sd_dreambooth_extension/wiki/Class-Explained page to elaborate on that section.

**Comment by felixsanz** [comment]
> Maybe I could add a line about this to the guide. The whole idea behind the `[filewords]` processing is that you can caption all of your images before you pick your instance and class tokens. Then when you actually run the extension, it will handle redundant words.

So... it just replaces (inside prompts) instance token and class token depending if the prompting is to generate the instance or the class?

**Comment by ArrowM** [comment]
> So... it just replaces (inside prompts) instance token and class token depending if the prompting is to generate the instance or the class?

basically yes (the actually implementation is slightly more sophisticated).

> Also if you're training Arnold Schwarzenegger, why use class "man" ? why not class "european white muscular man"? Of course model will be more rigid and you can't get an ethiopean arnold but what if that's fine?

`european white muscular man` would probably also be fine. I don't think it would make a huge difference, but I'm also not an expert. Try both or either :)
